pkg/controller/migplan/pvlist.go

```diff
@@ -115,3 +119,23 @@ func (r *ReconcileMigPlan) getClaims(client k8sclient.Client, namespaces []strin
 
 	return claims, nil
 }
+
+// Determine the supported PV actions.
+func (r *ReconcileMigPlan) getSupportedActions(pv core.PersistentVolume) []string {
+	if pv.Spec.HostPath != nil {
+		return []string{}
+	}
+	if pv.Spec.NFS != nil ||
+		pv.Spec.Glusterfs != nil ||
+		pv.Spec.AzureDisk != nil ||
+		pv.Spec.AzureFile != nil ||
+		pv.Spec.AWSElasticBlockStore != nil {
+		return []string{
+			migapi.PvCopyAction,
```

Comment:
This looks correct. Move and Copy are supported for NFS, Gluster, Azure, and EBS. What about GCP? Or do we only care about these for now?

Reply:
Only care about these for now.